api/v1beta1/index/cluster.go

```diff
@@ -55,3 +69,12 @@ func ClusterByClusterClassClassName(o client.Object) []string {
 	}
 	return nil
 }
+
+// ClusterByClusterClassClassNamespace contains the logic to index Clusters by ClusterClass namespace.
+func ClusterByClusterClassClassNamespace(o client.Object) []string {
```

Comment:
does this need the stutter, could we just name it `ClusterByClusterClassNamespace` ?

or if we need the extra context could we name it something different like `ClusterByClusterClassTopologyNamespace` ?

Reply:
Ah, good catch, I’ll update it. I think ClusterByClusterClassNamespace follows the naming pattern here.